notebooks/sentiment_analysis.py

```diff
@@ -0,0 +1,338 @@
+)
+out = newer_model(simple_test_data + ambiguous_prompts)
+# print prompts and predicted labels side by side
+pprint([*zip(simple_test_data + ambiguous_prompts, out)])
+
+"""
+This model performs similarly on these prompts as the model explored in the previous section. Differences in the model may not be apparent until we start prompting about recent events.
+
+If we ask the previous model to classify a statement about the Coronavirus pandemic we get different results between the models:
+"""
+coronavirus_prompt = ["Coronavirus is increasing"]
+old = tweet_model(coronavirus_prompt)
+new = newer_model(coronavirus_prompt)
+print(f"Older model score: {old}")
+print(f"Newer model score: {new}")
+
+"""
+The newer model has a strong negative connotation for Coronavirus while the older model sees it as a neutral statement. This simple experiment shows the importance of testing and fine-tuning models regularly to make sure that sentiment analysis continues to be accurate as connotations of certain words evolve.
+"""
+# We stop using these pipelines in the rest of the notebook so we detach from the
+# IPU devices to allow us to test additional models for more details see the section
+# on managing resources at the end of the document.
+tweet_model.model.detachFromDevice()
+newer_model.model.detachFromDevice()
+
+"""
+### Multi-lingual model
+
+The next model has an interesting feature: it is multi-lingual. It was trained on a dataset of English, Dutch, German, Spanish, Italian and French text, it can be prompted in any of these languages and should correctly classify the text inputs.
+
+The model is `nlptown/bert-base-multilingual-uncased-sentiment` a BERT checkpoint fine-tuned on ~700k reviews in 6 languages and predicts between 1 and 5 stars for each prompt.
+"""
+model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
+multilingual_model = pipelines.pipeline(
+    model=model_name, ipu_config_kwargs=inference_config
+)
+print(simple_test_data)
+multilingual_model(simple_test_data)
+
+"""
+It successfully classifies our simple input, now let's see how it fares with our ambiguous input:
+"""
+multilingual_model([
+    "How are you today?",
+    "I'm a little tired, I didn't sleep well, but I hope it gets better"
+])
+
+"""
+While it is a bit optimistic about our first prompt, it's guess is given with a fairly low confidence, and it identifies the second prompt as neutral with a median score of 3.
+
+Now let's translate our prompts and ask it the same questions in French:
+"""
+ambiguous_in_french = [
+    "Comment vas-tu aujourd'hui?",
+    "Je suis un peu fatigue, je n'ai pas bien dormi mais j'espere que la journee s'ameliore",
+]
+multilingual_model(ambiguous_in_french)
+
+
+"""
+The model gives very similar results in both languages!
+
+Now let us revisit our first model, could it also work with this multi-lingual input?
+"""
+print(sentiment_pipeline(ambiguous_prompts))
+print(sentiment_pipeline(ambiguous_in_french))
```

Comment:
Can you rename `sentiment_pipeline` at the start to `sentiment_model` to fit in with the same structure as the other models used in this tutorial?

Reply:
Renamed all to `pipeline`